src/components/pages/Registration.tsx

```diff
@@ -126,9 +118,9 @@ const ticketList: Ticket[] = [
     },
     link: 'https://usuc.yapsody.com/event/book/803521/31648557'
   }
-];
+];*/
 
-const Tickets = (props: any) => {
+/*const Tickets = (props: any) => {
```

Comment:
No commented code please. If you want to retain the code fragment then switch off this component using a flag.